sdp/processors/datasets/commoncrawl/__init__.py

```diff
@@ -0,0 +1 @@
+from .commoncrawl import  SplitByVttSentence
```

Comment:
please add copyright header here and in other new init files. 
we should have copyright header in every file, even if it is otherwise empty